### ISSUE TITLE
Add temporary files to technical appraisal

### ISSUE THREAD

**Issue by amhanson9** [uga-libraries/accessioning-scripts#17]
**Location:** format_analysis_functions.py > match_technical_appraisal() 

**Description:** Add temporary files as another category to technical appraisal. Temporary files include file names that start with either “~” or “.”, which are often but not always identified by FITS as unknown binary. The temporary files category has precedence over the format category but not the trash category. 

**Impact:** If this is implemented, it should be removed from the technical-appraisal-logs.py script.

**Comment by amhanson9** [comment]
@emkaser The general-aip script also tests for if a file ends with .tmp or is Thumbs.db as part of identifying temp files. Do you want me to add those criteria?

**Comment by emkaser** [comment]
@amhanson9 Yes, adding .tmp and Thumbs.db would be helpful. Thanks!

**Comment by amhanson9** [comment]
@emkaser This issue is done, and the changes merged into the milestone branch. I didn't edit the technical appraisal logs script, since you are the original author. Do you want to be the one that does that, maybe after you've tested my updates?

**Comment by emkaser** [comment]
@amhanson9 Yes, I can make the edits to technical-appraisal-logs.py - thanks!
